### DISCUSSION TITLE
Increase Height of Plain Description Field

### DISCUSSION THREAD

**Discussion by JEEVAN28** [YetiForceCompany/YetiForceCRM#14322] [How to] [answered]
Hello,
I need to increase Description Field Height in Calendar Activity Edit and Detail View to at least show 10 lines.
![image](https://user-images.githubusercontent.com/12723595/109940164-c9d6dc80-7cf7-11eb-952e-3ca0061bd950.png)

Manually I have increased height and need result as below
![image](https://user-images.githubusercontent.com/12723595/109940272-e3782400-7cf7-11eb-8443-04c9d5333aeb.png)

I tried making changes in layouts/basic/modules/vtiger/edit/field/text.tpl file where code as below
at line no 22
<textarea name="{$FIELD_MODEL->getFieldName()}" tabindex="{$FIELD_MODEL->getTabIndex()}"
**changed to **
<textarea rows="10" name="{$FIELD_MODEL->getFieldName()}" tabindex="{$FIELD_MODEL->getTabIndex()}"

but no luck

please assist how to do this.

Thanks
Jeevan Pendhari

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
Change the height with classes in fieldparams

**Reply by JEEVAN28** [reply]
THANKS, done

**Reply by Jianbin719** [reply]
could you please kindly guide me how to make it ?

**Reply by mariuszkrzaczkowski** [reply]
editWidth w fieldparams
